src/plugins/intel_cpu/tests/functional/single_layer_tests/group_convolution.cpp

```diff
@@ -199,6 +201,24 @@ class GroupConvolutionLayerCPUTest : public testing::WithParamInterface<groupCon
     }
 };
 
+using ExpectFailedGroupConvolutionLayerCPUTest = GroupConvolutionLayerCPUTest;
+
+TEST_P(ExpectFailedGroupConvolutionLayerCPUTest, CompareWithRefs) {
+    run();
+    if (isBias) {
+        checkBiasFusing(compiledModel);
+    }
+    static std::mutex gtest_mutex;
```

Comment:
Just curious, why do we need to deal with multithreading here and can't just use original model and CPU instance?

Reply:
there are some limits on *EXPECT_FATAL_FAILURE*, [gtest document](https://google.github.io/googletest/advanced.html#catching-failures)

Reply:
I move the multithreading logic to CPUTestBase, so that maybe other test cases can reuse it

Reply:
already not to use EXPECT_FATAL_FAILUR, so not exists multi thread issue